src/client/src/shell/notification/TradeNotification.stories.tsx

```diff
@@ -38,19 +32,32 @@ const Centered = styled('div')`
   display: flex;
   align-items: center;
   justify-content: center;
-  background-color: grey;
 `
 
 const NotificationContainer = styled('div')`
-  width: 360px;
+  width: 320px;
+  height: 120px;
 `
 
-stories.add('Default', () => (
-  <Story>
-    <Centered>
-      <NotificationContainer>
-        <TradeNotification {...props} />
-      </NotificationContainer>
-    </Centered>
-  </Story>
-))
+const getPropsByStatus:(status:TradeStatus)=>Props = (status)=>(
```

Comment:
Minor: Formatting

Reply:
The formatting is correct on my machine.